### DISCUSSION TITLE
Hi. I have a question!

### DISCUSSION THREAD

**Discussion by dududamiao** [GriefPrevention/GriefPrevention#691] [Q&A] [answered]
How to set up to player just can break the blocks claimeds by him, and cant break the blocks not claimed by nobody?

**Comment by RoboMWM** [comment, marked as answer]
See wiki for configuration. Survivalrequiringclaims

**Comment by dududamiao** [comment]
Thank you! I am brazilian, so i don't know if my english was correct, sorry.
